src/Microsoft.DotNet.Cli.Utils/StreamForwarder.cs

```diff
@@ -83,11 +83,6 @@ public void Read(TextReader reader)
 
         private void WriteBuilder()
         {
-            if (_builder.Length == 0)
```

Comment:
The builder is written out per line using WriteLine so that shouldn't be necessary.

Seems like there is an edge case here when you have nothing but newlines though that isn't accounted for properly.